python/Ganga/Core/GangaRepository/GangaRepository.py

```diff
@@ -36,7 +36,7 @@ def __str__(self):
 class InaccessibleObjectError(GangaException):
 
     def __init__(self, repo=None, id='', orig=None):
-        GangaException.__init__(self, "Inaccessible Object")
+        super(InaccessibleObjectError, self).__init__("Inaccessible Object: %s" % str(id))
```

Comment:
It's always been the case that `"%s"` will explicitly call `str()` on the argument. This is defined by the language.